bench/src/main/scala/org/http4s/bench/SchemeBench.scala

```diff
@@ -0,0 +1,22 @@
+package org.http4s
+package bench
+
+import org.openjdk.jmh.annotations._
+
+@Fork(2)
```

Comment:
This probably won't play nice with scala.js but I guess bench won't be supported regardles

Reply:
Right.  We also don't run them as part of any build, just compile them.  I haven't figured out "benchmarking as a free service for open source" yet.